FSharpBuild.Directory.Build.props

```diff
@@ -87,4 +87,13 @@
     <FSharpOverridesTargetsShim>$(ProtoOutputPath)\fsc\Microsoft.FSharp.Overrides.NetSdk.targets</FSharpOverridesTargetsShim>
   </PropertyGroup>
 
+  <!-- use this to enable good debugging in the Proto bootstrap compiler -->
+  <!-- You'll have to remove it to get the actual compiler bo build because optimization is needed for some large methods -->
+  <!--
+  <PropertyGroup Condition="'$(Configuration)' == 'Proto'">
+    <DebugSymbols>true</DebugSymbols>
+    <Optimize>false</Optimize>
```

Comment:
We sure we want to do this? I'm used to knowing that the proto compiler is the fully optimized compiler. What scenario requires debugging the proto compiler?

Reply:
It's disabled by default, I enable it when the bootstrap compiler we build no longer works due to changes in the compiler, i..e when you hork the bootstrap